<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandMasterGUI</class>
 <widget class="QMainWindow" name="BandMasterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BandMaster</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" rowspan="3">
     <widget class="QFrame" name="PSGenFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="PSGen_label">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Probe Signal Generator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="PSGenParamFrame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="pulseDurationLabel">
            <property name="text">
             <string>Duration, msec</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="pulseDurationSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="minimum">
             <number>50</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="PSFreq1Label">
            <property name="text">
             <string>Frequency 1, Hz</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="PSFreqSpinBox1">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>Hz</string>
            </property>
            <property name="minimum">
             <number>60</number>
            </property>
            <property name="maximum">
             <number>8000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>4200</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="PSFreq2Label">
            <property name="text">
             <string>Frequency 2, Hz</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="PSFreqSpinBox2">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>Hz</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>8000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="PSFreq3Label">
            <property name="text">
             <string>Frequency 3, Hz</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="PSFreqSpinBox3">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix">
             <string>Hz</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>8000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PSGen_pushButton">
         <property name="text">
          <string>Generate probe signal file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QFrame" name="sourcesFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="sourcesLabel">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Channel sources</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ch1label">
         <property name="text">
          <string>ch 1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="ch1combo"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="ch2combo"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ch2label">
         <property name="text">
          <string>ch 2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="ch4label">
         <property name="text">
          <string>ch 3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="ch3combo"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="ch4combo"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="ch4label_2">
         <property name="text">
          <string>ch 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QFrame" name="paramFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="5" column="0">
        <widget class="QLabel" name="timerLabel">
         <property name="text">
          <string>Record time, sec</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="timerSpinBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="durationLabel">
         <property name="text">
          <string>File duration, sec</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="fileDurationSpinBox">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="sampleRateLabel">
         <property name="text">
          <string>Sample rate, Hz</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="sampleRateBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="bitLabel">
         <property name="text">
          <string>Format, bit</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="bitFormatBox"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="paramLabel">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Record parameters</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="fileFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>File ID (Project)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="folderFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="folderLabel">
         <property name="text">
          <string>Folder</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="folderButton">
         <property name="text">
          <string>Select work folder</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="folderPathLabel">
         <property name="text">
          <string>Path</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" rowspan="3">
     <widget class="QFrame" name="PSPlayerFrame">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="2" column="0" colspan="2">
        <widget class="QLineEdit" name="PSFileLineEdit1">
         <property name="text">
          <string>File_name_1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="PSFileSelectButton1">
         <property name="text">
          <string>Select file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="PSigCheckBox1">
         <property name="text">
          <string>Signal 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="PSPlayerLabel">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Probe Signal Player </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="PSLevelSpinBox1">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSlider" name="PSLevelHorizontalSlider1">
         <property name="maximumSize">
          <size>
           <width>84</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="pulsePeriodLabel_1">
         <property name="text">
          <string>Period 1, sec</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="PSPeriodSpinBox1">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>s</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="PSigCheckBox2">
         <property name="text">
          <string>Signal 2</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="PSFileSelectButton2">
         <property name="text">
          <string>Select file</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLineEdit" name="PSFileLineEdit2">
         <property name="text">
          <string>File_name_2</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="PSLevelSpinBox2">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QSlider" name="PSLevelHorizontalSlider2">
         <property name="maximumSize">
          <size>
           <width>84</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="PSPeriodSpinBox2">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="suffix">
          <string>s</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="pulsePeriodLabel_2">
         <property name="text">
          <string>Period 2, sec</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="outDeviceLabel">
         <property name="text">
          <string>Output device</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QComboBox" name="outputComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QFrame" name="actionFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>BandMasterGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>841</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ch1combo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BandMasterGUI</receiver>
   <slot>slotInpSourceChg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ch2combo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BandMasterGUI</receiver>
   <slot>slotInpSourceChg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ch3combo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BandMasterGUI</receiver>
   <slot>slotInpSourceChg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ch4combo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>BandMasterGUI</receiver>
   <slot>slotInpSourceChg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>folderButton</sender>
   <signal>clicked()</signal>
   <receiver>BandMasterGUI</receiver>
   <slot>slotChooseFolder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PSLevelSpinBox1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PSLevelHorizontalSlider1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PSLevelHorizontalSlider1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PSLevelSpinBox1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PSLevelSpinBox2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PSLevelHorizontalSlider2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PSLevelHorizontalSlider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PSLevelSpinBox2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>BandMasterGUI</receiver>
   <slot>slotToggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>829</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>setFolder(QString)</signal>
  <slot>slotChooseFolder()</slot>
  <slot>slotInpSourceChg()</slot>
  <slot>slotToggle()</slot>
 </slots>
</ui>
